test/e2e/common/node/container_probe.go

```diff
@@ -520,7 +520,8 @@ var _ = SIGDescribe("Probing container", func() {
 		Description: A Pod is created with liveness probe on grpc service. Liveness probe on this endpoint will not fail. When liveness probe does not fail then the restart count MUST remain zero.
 	*/
 	ginkgo.It("should *not* be restarted with a GRPC liveness probe [NodeAlphaFeature:GRPCContainerProbe][Feature:GRPCContainerProbe]", func() {
-		e2eskipper.SkipUnlessFeatureGateEnabled(kubefeatures.GRPCContainerProbe)
+		// TODO(SergeyKanzhelev): it is unclear when feature gates are not working as expected.
+		//e2eskipper.SkipUnlessFeatureGateEnabled(kubefeatures.GRPCContainerProbe)
```

Comment:
feature gates , as we are used to them on kubelet and other components, doesn't apply to the e2e framework, is a differnt binary, I think that you can use them, but you have to pass them as a config of flag option before running the test

Reply:
I understand this. Test binary also reads them from test options. You may see it is working sometimes: https://testgrid.k8s.io/sig-node-kubelet#node-kubelet-alpha&include-filter-by-regex=GRPC So I hunted it down, but got stuck and for now want to unbock the PR.

This test has Feature tag that will not let it run under any lanes, so this is safe to comment out (but not ideal)